browser/ui/views/split_view/split_view_browsertest.cc

```diff
@@ -58,7 +59,8 @@ class SideBySideEnabledBrowserTest : public InProcessBrowserTest {
  public:
   SideBySideEnabledBrowserTest() {
     scoped_features_.InitWithFeatures(
-        /*enabled_features*/ {features::kSideBySide}, {});
+        /*enabled_features*/ {features::kSideBySide},
+        /*disabled_features*/ {features::kBraveWebViewRoundedCorners});
```

Comment:
why do we need to disable rounded corners?

Reply:
When rounded corners are enabled, that separator is always hidden.